src/AbstractResponse.php

```diff
@@ -105,9 +105,10 @@ public function setStatusCode($statusCode)
      * @param callable|\League\Fractal\TransformerAbstract $transformer
      * @param string $resourceKey
      * @param array $meta
+     * @param array $headers
      * @return mixed
      */
-    public function withItem($data, $transformer, $resourceKey = null, $meta = [])
+    public function withItem($data, $transformer, $resourceKey = null, $meta = [], array $headers = array())
```

Comment:
Can you replace `array $headers = array()` by `array $headers = []` please. The package only support php 5.4 and greater, so you should use the new array syntax.